Prace_domowe/Praca_domowa2/Grupa2/Paweł_Fijałkowski/PD2.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<p>To trochę krótko jak na odpowiedź na pytanie "Czy i jakie są przewagi target encoding nad one-hot?". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>